node/coinstacks/base/api/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@shapeshiftoss/base-api",
+  "version": "10.0.0",
+  "license": "MIT",
+  "private": true,
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn generate && tsc",
+    "clean": "rm -rf dist node_modules src/routes.ts src/swagger.json",
+    "generate": "tsoa spec-and-routes"
+  },
+  "dependencies": {
+    "@eth-optimism/contracts-bedrock": "^0.14.0",
```

Comment:
update to use base contracts

Reply:
updated to new ts package and contracts are deployed at the same address on both optimism and base